Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6317,7 +6303,7 @@ SpeechRecognitionEnabled:
 
 SpeechSynthesisAPIEnabled:
   type: bool
-  status: embedder
```

Comment:
I wouldn't mess with that in the same PR, can you undo this ?

Reply:
Sure, I can make that a separate PR.